luxonis_train/core/trainer.py

```diff
@@ -74,6 +77,7 @@ def graceful_exit(signum: int, _):
                     remote_path="resume.ckpt",
                     mlflow_instance=self.tracker.experiment.get("mlflow", None),
                 )
+                mlflow.end_run(RunStatus.to_string(RunStatus.FAILED))
```

Comment:
We use two trackers in tuner with optuna. How does this work in such case? CC @klemen1999

Reply:
I think the ending of tracker runs should be handled inside `LuxonisTrackerPL` since base PL Logger has `finalize` method that is called on every teardown with status. I'll add this to the #26 and you should be then able to remove this line from here. 
Regarding 2 trackers in Tuner: MLFlow uses stack for active trackers so it would end them in reverse order that they are started which works fine

Reply:
Ok, sounds good. Let's add to #26. And we can then remove @kozlov721 here.